src/components/sidebar/Sidebar.tsx

```diff
@@ -61,7 +62,7 @@ export function Sidebar({
         <Inner isOpen={isOpen} {...props}>
           {isOpen && (
             <>
-              <Header reverse={side === "onLeft"}>
+              <Header reverse={side === "onLeft"} padding={padding}>
```

Comment:
I think we could allow passing `headerProps` & `containerProps` (not just padding), this will give us freedom to fully customize those 2 containers.